examples/pixelcnn/README.md

```diff
@@ -26,11 +26,18 @@ The model should run with other configurations and hardware, but was tested on t
 
 ### How to run
 #### 8 x Nvidia V100 (16GB), 8 x TPUv3 (16GB)
-To run training
+
+To run training:
+
 ```
```

Comment:
Nit: maybe add `shell` and then Markdown rendering can add fancy colors

```suggestion
```shell
```